src/main/java/Oley.java

```diff
@@ -12,43 +35,64 @@ public static void initialise(){
         System.out.println("    Hello, I'm your cute and lovely friend Oley.");
         System.out.println("    What can I do for you?");
         lineBreaker();
-
     }
-    public static void exist(){
+    public static void exit(){
         System.out.println("    Bye~ Feel free to talk to me anytime. I will always be here waiting for you. ฅʕ•̫͡•");
     }
     public static void lineBreaker(){
         System.out.println(" ");
         System.out.println("    🌙~~~~~♥~~~~~♥~~~~~♥~~~~~♥~~~~~♥~~~~~♥~~~~~🌙");
     }
 
+    public static void printTask(){
+        System.out.println("    Here are the tasks in your list:");
+        int i = 0;
+        while (i < taskNumber) {
+            System.out.println("    " + (i + 1) + "." + tasks[i].toString());
+            i++;
+        }
+    }
+
+    public static void mark(String sentence){
+        String[] markInstructions =  sentence.split(" ");
+        int toBeMarked = Integer.parseInt(markInstructions[1]) - 1;
+        if (toBeMarked >= taskNumber){
+            System.out.println("You have not created Task " + (toBeMarked + 1) + " yet. Jiayous. I will always support you. ฅ •ﻌ•♡");
+        }
```

Comment:
Perhaps you could place the elses on the same line as the ending bracket of the if to better align with the standards